packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,26 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private getTitle() {
+        // Either get the title from the document, or return the default title
+        let title = "Blueprint - Documentation";
+        const docsTitle = document.getElementsByClassName("docs-title")[0].textContent;
+        // Check to make sure the textContent of docsTitle has a useable value
+        if (docsTitle !== undefined && docsTitle !== null) {
+            // Only change the title if we aren't on the index homepage
+            if (this.state.activePageId !== "blueprint" && this.state.activeSectionId !== "blueprint") {
+                title = `${docsTitle} - Blueprint`;
```

Comment:
I find it awkward that the word "Blueprint" moves from the beginning to the end based on page. I'd prefer "Blueprint" always appear in the same position on all pages.

Reply:
tabs can only show a limited amount of characters, so I think it's more useful to show the page/component name first. it's actually quite a common pattern across the web

Reply:
also nit pick, would you mind changing the regular dash `-` to an em-dash `—`? anybody correct me if I'm wrong, but according to Google this usage calls for the em-dash